### DISCUSSION TITLE
NLA (Network Level Authentication) support request

### DISCUSSION THREAD

**Discussion by GEOEGII555** [neutrinolabs/xrdp#2772] [Q&A]
When will we get NLA support? It was been promised 8-9 years ago but we still didn't get it: https://xrdp-devel.narkive.com/vOblisLe/how-to-enable-network-level-authentication-in-xrdp

**Comment by matt335672** [comment]
Not any time soon I'm afraid.

It's a very different beast from TLS - there's a good overview [here](https://syfuhs.net/how-authentication-works-when-you-use-remote-desktop) on Steve Syfuhs' blog. In summary there are four scenarios which would need to be supported, two of which (NTLM and AAD) which will be challenging, to say the least.
